src/chrome/content/rules/SaltStack.com.xml

```diff
@@ -11,9 +11,13 @@
 -->
 <ruleset name="SaltStack.com">
 	<target host="saltstack.com" />
+	<target host="bootstrap.saltstack.com" />
 	<target host="docs.saltstack.com" />
+	<target host="repo.saltstack.com" />
 	<target host="www.saltstack.com" />
```

Comment:
Please sort the `www` subdomain below the root domain, as defined [here](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#target-ordering).